examples/bzlmod/.bazelrc

```diff
@@ -1 +1,3 @@
 common --experimental_enable_bzlmod
+
+coverage --java_runtime_version=remotejdk_11
```

Comment:
My guess would be because they don't have java installed on their dev machine, or at least not in a way that bazel is able to find it, and including it in the PR was accidental.

Reply:
Should I remove it? I installed `bazel` via `bazelisk` and it does not pull in any `java` dependencies by default. What is the official way to do this?

Reply:
So according to this: https://bazel.build/docs/bazel-and-java#hermetic-testing

What this basically means is instead of using a locally installed JDK, it uses one it downloads (it sounds similar to how we're doing things).

So this is probably a net positive for getting started at the cost of having to download a JDK. I think that's worth it, so go ahead and leave it in.